src/dms/core/EdFi.DataManagementService.Core.Tests.Unit/Handler/DeleteByIdHandlerTests.cs

```diff
@@ -64,7 +64,7 @@ public override Task<DeleteResult> DeleteDocumentById(IDeleteRequest deleteReque
             }
         }
 
-        private readonly RequestData _context = No.RequestData();
+        private readonly RequestInfo _requestInfo = No.RequestInfo();
```

Comment:
I went and renamed RequestData (formerly PipelineContext) one last time to RequestInfo. It matches well with ResourceInfo and DocumentInfo being derived from it.